OutOfSchool/OutOfSchool.EmailSender/EmailOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace OutOfSchool.EmailSender
+{
+    public class EmailOptions
+    {
+        public static string SectionName => "Email";
+
```

Comment:
inconsistent whitespace))

Reply:
fixed and also fixed the same in SmtpOptions